docs/API.md

```diff
@@ -60,7 +60,7 @@ for the specified method is invalid, an error with code `-32602` is reported.
 For more information on the validation error data format see
 [Validation error data].
 
-All error states that occur after the RPC method has been identified are reported as internal errors with code `-32603`.
+All errors states that occur after the RPC method has been identified are reported as internal errors with code `-32603`.
```

Comment:
I think that "error states" is more correct.

Reply:
oops, I have replaced  too much.